### DISCUSSION TITLE
Python Vs R

### DISCUSSION THREAD

**Discussion by IshikaMahajan7** [community/community#41785] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![1_DdGKJSjwI928HgP4NC57Kw](https://user-images.githubusercontent.com/115925848/207750114-58512234-665a-4dcd-9ef2-84cc4da1baa8.jpeg)

which one in better for future

**Comment by sagarbangade** [comment, marked as answer]
Python is better than R as it can be used for multiple purposes. It has better scalability, performance, integration, etc. However, if the purpose is data analysis and visualization, R is a better option.
